dll/win32/shell32/wine/pidl.c

```diff
@@ -476,6 +476,7 @@ LPITEMIDLIST WINAPI ILGlobalClone(LPCITEMIDLIST pidl)
     return newpidl;
 }
 
+#ifndef __REACTOS__
 BOOL _ILHACKCompareSimpleIds(LPCITEMIDLIST pidltemp1, LPCITEMIDLIST pidltemp2)
```

Comment:
This "HACKCompare..." function was introduced there: https://github.com/reactos/reactos/commit/2af89ef8f01423c913f47df6e321cc1cec07b3bc
by @yagoulas . So it doesn't need to be "ifndef __REACTOS__"'ed.